# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../djv/djv-0.8.2-rename_test_audio.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Rename rename audio test tool 'test_audio' to 'djv_test_audio'

--- djv-0.8.2/tests/audio/CMakeLists.txt.orig	2009-07-18 21:36:30.000000000 +0200
+++ djv-0.8.2/tests/audio/CMakeLists.txt	2011-04-15 17:31:18.780078685 +0200
@@ -7,29 +7,29 @@
   ${QUICKTIME_INCLUDE_DIR}
 )
 
-add_executable(test_audio ${header} ${src})
+add_executable(djv_test_audio ${header} ${src})
 
-add_dependencies(test_audio ${djv_ui_dep})
+add_dependencies(djv_test_audio ${djv_ui_dep})
 
 target_link_libraries(
-  test_audio
+  djv_test_audio
   ${djv_ui_lib}
   ${QUICKTIME_LIBRARIES}
   ${PORTAUDIO_LIBRARIES}
 )
 
-target_link_libraries(test_audio ${djv_ui_lib})
+target_link_libraries(djv_test_audio ${djv_ui_lib})
 
 if (djv_install_rpath)
   set_target_properties(
-    test_audio PROPERTIES
+    djv_test_audio PROPERTIES
     INSTALL_RPATH "${djv_install_rpath}"
   )
 endif (djv_install_rpath)
 
-install(TARGETS test_audio RUNTIME DESTINATION bin)
+install(TARGETS djv_test_audio RUNTIME DESTINATION bin)
 
 if (APPLE)
-  DJV_OSX_INSTALL_BIN(test_audio)
+  DJV_OSX_INSTALL_BIN(djv_test_audio)
 endif (APPLE)
 
